src/components/dashboard-create/dashboard-create.tmpl.html

```diff
@@ -0,0 +1,6 @@
+<div class="dashboard-create">
+  <button class="btn" ng-class="$ctrl.btnClass" ng-click="$ctrl.openModal()">
```

Comment:
`.btn` class should not be used alone, but coupled with a semantic class (eg. `.btn-primary`)

Reply:
yeah, I've just made it possible to customise the following `btn-something` by using: `ng-class="$ctrl.btnClass"`: this way I can use the same component to display normal or large buttons. In the admin panel, this attribute is used to display the component as a primary button... So in the end it will have `.btn` + whatever is passed to `$ctrl.btnClass`